kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/HtmlPageFilter.java

```diff
@@ -110,7 +121,15 @@ public ContentResource wrap(final ContentResource content) {
 			String docType = serverConfigurationService.getString("content.html.doctype", "<!DOCTYPE html>");
 			header.append(docType + "\n");
 		}
-		header.append(MessageFormat.format(headerTemplate, skinRepo, siteSkin, title));
+		StringBuilder additionalScripts = new StringBuilder();
+		if (isMathJaxEnabled(entity)) {
+			additionalScripts.append(MessageFormat.format(mathjaxTemplate,
+				serverConfigurationService.getString(MATHJAX_SRC_PATH_SAKAI_PROP,
+					"https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=default,Safe"
```

Comment:
I don't think this URL for MathJax should be hardcoded, as it can be overridden in sakai.properties. See:

https://github.com/sakaiproject/sakai/pull/3007/files